examples/apps/cra-demo/README.md

```diff
@@ -1,210 +1,240 @@
 
-    const load = async () => {
-        const fluidContainer = isNew
-            ? await Fluid.createContainer(containerId, [KeyValueDataObject])
-            : await Fluid.getContainer(containerId, [KeyValueDataObject]);
+### 4.a Create a place to store state
 
-        const keyValueDataObject: KeyValueDataObject = isNew
-            ? await fluidContainer.createDataObject(KeyValueDataObject, 'kvpairId')
-            : await fluidContainer.getDataObject('kvpairId');
+In our demo, we'll be storing a single date value in our `SharedMap` and displaying that value in our view. Since changes to the `SharedMap` won't trigger an update in our React view, we need to store the view's `date` in React state and then modify that state each time our view should update via `setDate`.
 
-        setDataObject(keyValueDataObject);
-    }
+We will also need to store the `SharedMap` because it will only be created after we create the Fluid container.
 
-    load();
+```jsx
+function App() {
+    const [date, setDate] = React.useState('');
+    const [map, setMap] = React.useState();
 
-}, [])
+    // return(...)
+};
 ```
 
-Once the DataObject is returned we assign it to our `dataObject` state variable, and now we have access to all of the KVPair's methods, including `set` which we pass down as the `setPair` function.
+Now that we have state, we need a way to update that state when our React component loads. That's where we turn to `useEffect`.
 
-Here's our hook so far.
+### 4.b Load container and subscribe to changes to Fluid data
 
+React's `useEffect` takes a function as its first parameter that fires as soon as the component loads and then fires again if any of its dependencies change (in our case the `map` state variable, specified in the second parameter).
 
-```tsx
-function useKVPair(): [KVData, SetKVPair | undefined] {
-    const [dataObject, setDataObject] = React.useState<KeyValueDataObject>();
+Our `useEffect` will end up firing twice. The first time, on component load, it will either create or load the container depending on if it's a new 'document'.
 
-    React.useEffect(() => {
-        const { containerId, isNew } = getContainerId();
-
-        const load = async () => {...
-        }
+The second time `useEffect` fires, the `map` will have been set and we can set up a listener to update `date` each time the `valueChanged` event is fired.
 
-        load();
+```jsx
+function App() {
 
-    }, [])
+    const [map, setMap] = React.useState();
+    const [time, setTime] = React.useState('');
 
-    const setPair = dataObject?.set;
+    React.useEffect(() => {
+        if (!map) {
+            // First time: create/get the Fluid container
+            const { containerId, isNew } = getContainerId();
+            const containerConfig = {
+                name: 'cra-demo-container',
+                initialObjects: { map: SharedMap }
+            };
+
+            const load = async () => {
```

Comment:
I think this code might need to get updated to reflect the changes in the App.js?
i.e.  `await TinyliciousClient.createContainer({ id }, containerConfig)` where `id` is in an object, `setData` instead of `setMap` etc.

Reply:
Yeah, sorry...Readme is not updated yet